guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -194,29 +171,24 @@ In the following example `/root` is the extraction location.
 +
 [options="nowrap" subs="attributes"]
 ----
-[{RepoRHEL7ServerAnsible}]
-name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerAnsible}
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=file:///root/{Project}-repos/{RepoRHEL8BaseOS}
 enabled=1
 
-[{RepoRHEL7Server}]
-name=Red Hat Enterprise Linux 7 Server RPMs x86_64
-baseurl=file:///root/{Project}-repos/{RepoRHEL7Server}
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={ProjectNameX} for RHEL 8 Server RPMs x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL8ServerSatelliteServerProductVersion}
 enabled=1
 
-[{RepoRHEL7ServerSatelliteServerProductVersion}]
-name={ProjectNameX} for RHEL 7 Server RPMs x86_64
-baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSatelliteServerProductVersion}
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance 6 for RHEL 8 Server RPMs x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL8ServerSatelliteMaintenanceProductVersion}
 enabled=1
 
-[{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
-name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
-baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSatelliteMaintenanceProductVersion}
-enabled=1
-
-[{RepoRHEL7ServerSoftwareCollections}]
-name=Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSoftwareCollections}
+[{RepoRHEL8AppStream}]
+name=Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
```

Comment:
```suggestion
name=Red Hat Enterprise Linux 8 for x86_64 {endash} AppStream (RPMs)
```

Is that correct @Lennonka ?
See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-typography.adoc

Reply:
no, please don't add funny unicode to these files

Reply:
Indeed. Adding unicode into code blocks could potentially result in the commands / files being unusable.

Reply:
Oh, good call!